### DISCUSSION TITLE
Database for LAGUNA III EnergydCi (175ch) eco2

### DISCUSSION THREAD

**Discussion by djnaoki-mrn** [cedricp/ddt4all#1187] [Q&A]
Hello,
i would like to add a rear view camera to my grandpa's car wich is a Renault  LAGUNA III EnergydCi (175ch) eco2.
here you can find a short documentation https://www.type-mine.com/136055/renault-laguna-iii-energydci-175ch-eco2.html

the fact is that on this car you only can acces to the Rlink 1 parameters from the ODB connector so i need DDT4ALL to enable the rear cam option.

now i have 2 issues : 
first i have 2 OBD adapters 1 OBDII wifi with PIC18F25K80 and 1 ELS27 with PIC24 + FT232RQ. when i use an old version (like 2019) of DDT4ALL i can connect my OBDII without any issue but with the newer version i only have a message saying that the software try to connect over the network. i tryed to ping the adapter and everything is ok. if i start DDT4all (newest version), and try to connect and start an other software that use this adapter, DDT4all the initialize the conenction but everything is buggy.

secondly when i use the old version of DDT4ALL with my database i can found the laguna III but it seems that tihs model in the database is not compatible with the car. then it seems that the car never reply to DDT4all. All parameters stey in ''NO DATA'' state.
When i use the magnifying glass button, some parameters appears in green and then i'm able to read data.

is somwone as any idea of whats appening ?

ps: i've tried to benchmark my to adapters on the welcome page of the software (TEST ELM button). and everythig seems to be good. i've got no incompatibility message.

thank you. have a good day.

**Comment by AndyIreland** [comment]
Hi

is it definitely a rlink1?

- Have you specially wired up a OBD2 splitter cable so you can read the rlink through the cars OBD socket?

**Reply by djnaoki-mrn** [reply]
Hello, thank you for your fast reply.
Yep i'm completely sure that's a rlink 1. 
What do you mean by a ''splitter'' ? 
Did you have a pinout shematics of the cable ?
If I understand correctly what you meant, the rlink is not wired ? 

Thank you

**Comment by cedricp** [comment]
Multimedia CAN is not on the same OBD pins than Vehicle CAN, there are 2 separate networks. see #512

**Comment by AndyIreland** [comment]
here is a wiring diagram in this article how to modify for reading the rlink through the OBD2 socket:

https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/

Rlink runs on CAN2 network - all other ECU's run on CAN1 (or CAN-L)

**Reply by djnaoki-mrn** [reply]
thank you. i will try this and let you know soon.

**Comment by djnaoki-mrn** [comment]
hi everybody. i've made the modification on the odb2 wifi adapter and it works pretty well !
the camera is working fine now. i only have one issue : i have a message about the ''non official guide line''. is somone know somthing aout that and how to remove this message ? 
i've tryed to make a manual calibration on the guideline trhougth ddt4all. it seems that the calibration worked but the message still there. 

thank you !

**Comment by djnaoki-mrn** [comment]
![IMG_20240807_190906](https://github.com/user-attachments/assets/e826ccb3-4b7a-4641-b040-2fb933c13a35)

**Reply by NiiHam** [reply]
Hi djnaoki-mrn,

I am also facing a similar problem connecting my Laguna 3 2013 gt. I did the connector re pinning but still no successful connection with ELM 327 and OBDlink SX. I have been able to read the ECU CAN however with the OBDwiz software using OBDLink SX. Probably my ELM327 may be faulty. Is there a way to go through the whole process using the OBDLink SX dongle with ddt4all instead of the ELM327?

**Comment by mishu28nmv** [comment]
Did you do the settings like in the picture? I mean the settings for the camera?
just the ones with:
"Rear cam dynamic guidelines=**Indication**"
"Rear cam fixed guidelines=**Indication**"
"Rear cam guidelines setting=**Default guidelines set (Default value)**"
[R-Link_1 Camera Settings in ddt4all](https://drive.google.com/file/d/1QzX-ZBbfLWM8r7yXj8x2QKwt2YCOzUUT/view)
